### PR TITLE
Removing an example of htpasswd_idp password due to an alert from the…

### PR DESCRIPTION
**Pull request by bardielle**
… scanner

### CODE REVIEW

File: docs/resources/identity_provider.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ resource "rhcs_identity_provider" "htpasswd_idp" {
   htpasswd = {
     users = [{
       username = "user1"
```
Comment (nirarg) [inline docs/resources/identity_provider.md:22]:
```suggestion
      username = "<username>"
```

**Comment by sagidayan** [comment]
did not understand the issue... plus no ticket attached

That being said, don't see the issue with this PR. 
/approve

**Pull request merged**